app/errors/liquid_tags.rb

```diff
@@ -0,0 +1,9 @@
+module LiquidTags
+  module Errors
+    class Error < StandardError
+    end
+
+    class InvalidParsedContext < Error
+    end
```

Comment:
Can you add a comment to explain withat `invalid parsed context` applies to? I can see it by reading the PR but maybe an explanation is useful as well